symplectic_manifolds.tex

```diff
@@ -1,35 +1,33 @@
+\documentclass[main.tex]{subfiles}
+\begin{document}
+
 \chapter{Symplectic geometry}
-\section{Symplectic vector spaces}
+Symplectic and Poisson structures came from the structures observed in analytical mechanics (originated from the need of studying complex dynamical systems in exact fashion, like motions of planets, oscillations of many particles systems, etc.).
+\section{Symplectic forms on vectors spaces}
+\subsection{Skew-symmetric bilinear maps}
 \begin{definition}
-	A \textbf{symplectic form} is a non-degenerate, skew-symmetric bilinear form. A vector space $V$ equipped with a symplectic form on $V$ is called a \textbf{symplectic vector field}.
+	Let $V$ be an $\R$-vector space with $\dim V=m$. Let $\Omega:V\times V\to\R$ be a bilinear map. $\Omega$ is \textbf{skew-symmetric} iff $\Omega(v,w)=-\Omega(w,v)$ $\forall v,w\in V$.
 \end{definition}
 
-Notice the analogy to an inner-product space, where skew-symmetry of the form is replaced by simple symmetry. Indeed, some results generalize as they depend only on the non-degeneracy of the bilinear form, for instance partial application of the form $\Omega$ fixes a canonical isomorphism of $V$ and its dual:
-\begin{eqalign}
-	\tilde\Omega : V &\longto V^*\\
-	v &\longmapsto \Omega(v, -);
-\end{eqalign}
-On the other hand, the theory of inner product spaces and that of symplectic spaces start to diverge very early on:
-
 \begin{theorem}
 \label{th:decomp_th}
-	Let $V$ be a finite-dimensional $\R$-vector space. Let $\Omega$ be a skew-symmetric, bilinear map on $V$. Then there exists a basis
+	Let $V$ be a finite-dimensional $\R$-vector space. If $\Omega:V\times V\to\R$ is skew-symmetric then there exists a basis
 	\begin{eqalign}
 		B= \{ u_1, \ldots, u_k, e_1, \ldots, e_n, f_1, \ldots, f_n \}
 	\end{eqalign}
-	such that
+	of $V$ such that
 	\begin{enumerate}
-		\item $\Omega(u_i, v) = 0$ for any $v \in V$ and $i \leq k$,
-		\item $\Omega(e_i, e_j) = \Omega(f_i, f_j) = 0$ for any $i, j \leq n$,
-		\item $\Omega(e_i, f_j) = \delta^i_j$ for any $i, j \leq n$,
+		\item $\Omega(u_i, v) = 0$ $\quad\forall\,v \in V$, $\forall\,i=1,\dots,k$,
+		\item $\Omega(e_i, e_j) = \Omega(f_i, f_j) = 0$  $\quad\forall\,i, j=1,\dots,n$,
+		\item $\Omega(e_i, f_j) = \delta_{ij}$ $\quad\forall\,i, j=1,\dots,n$,
 	\end{enumerate}
 \end{theorem}
 \begin{proof}
 	Let
 	\begin{eqalign}
-		U = \{ u \in V \suchthat \Omega(u,v) = 0\ \forall v \in V\}.
+		U := \{ u \in V \suchthat \Omega(u,v) = 0\ \forall\, v \in V\}.
```

Comment:
There's no space after `\forall`.
Revert every `\forall\,` to `\forall`, please.